File: Python3/2.py
#!/usr/bin/python3
sum_ = 2
a = 1
b = 2
c = a + b
while (c < 4_000_000):
	if c % 2 == 0:
		sum_ += c
	a, b = b, c
	c = a + b
print(sum_)

File: cpp/2/2.cpp
#include <iostream>
using namespace std;

int main()
{
	int a;
	int b;
	int c;
	int sum;

	a = 1;
	b = 2;
	c = a + b;
	sum = 2;
	while (c < 4000000)
	{
		if (c % 2 == 0)
			sum += c;
		a = b;
		b = c;
		c = a + b;
	}

	cout << sum << endl;
	
	return (0);
}